.github/workflows/build-macos.yaml

```diff
@@ -35,10 +35,10 @@ jobs:
 
       - name: Determine Toolchain Versions and cache paths
         run: |
-          echo NODE_VERSION=$(make -C build.assets print-node-version) >> $GITHUB_ENV
-          echo GOLANG_VERSION=$(make -C build.assets print-go-version | sed 's/^go//') >> $GITHUB_ENV
-          echo RUST_VERSION=$(make -C build.assets print-rust-version) >> $GITHUB_ENV
-          echo WASM_PACK_VERSION=$(make -C build.assets print-wasm-pack-version) >> $GITHUB_ENV
+          echo NODE_VERSION=$(make -s -C build.assets print-node-version) >> $GITHUB_ENV
```

Comment:
You should also add `--no-print-directory` like it has in the build-windows workflow if you want to stop the "Entering directory..." messages.

Reply:
It looks like `-s` includes `--no-print-directory` by default:

> As an aside, notice that the --no-print-directory option is automatically activated if the -s flag is used.

https://www.gnu.org/software/automake/manual/html_node/Tricks-For-Silencing-Make.html

Reply:
Good to know. It would be nice if the man page said that too, but I know how much GNU dislike man pages and prefer their info format instead. But even the info document index does not have either `-s` or `--no-print-directory`.